app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,26 @@
+.footer {
+  background: rgb(20,127,251);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 100px;
+  padding: 0px 50px;
+  color: rgb(255, 255, 255);
+}
```

Comment:
Right now footer appears right after the content, which makes it a little bit awkward (if there aren't enough content we will have the footer in the middle of the screen.

We can fix this by this command:
` position: fixed;
  bottom: 0;`

can you please add that and make another commit